spec/services/metrc_service/plant/discard_spec.rb

```diff
@@ -249,7 +249,7 @@
         expect(payload.first).to eq(
           Id: nil,
           Label: '1A4FF01000000220000010',
-          ReasonNote: 'Does not meet internal QC',
+          ReasonNote: '',
```

Comment:
Are we sure the API accepts an empty value? Did you test this payload against Metrc's sandbox?

Reply:
Good catch - I have tested and it does not. Updated with a generic default message of `Not Specified`.